src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -279,7 +279,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600]
   },
   unfollowedHeaderContainer: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the background color of `unfollowedHeaderContainer` might affect the visibility of any components inside this container. Make sure that the color changes are in line with the overall theme and do not affect the user experience negatively.

```diff
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: COLORS.gray100
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->